active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,150 @@
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+## Create a Breadcrumb
+
+![Breadcrumb default with 3 nodes](images/Breadcrumb_default.PNG)
+ ~~~~ 
+     <muxc: Breadcrumb ItemsSource = {"Node", "Node", "Node"} /> 
+~~~~
+More examples to be added
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+# API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+| Name | Description | Default |
+| :---------- | :------- | :------- |
+| ItemsSource | provides the content of the Breadcrumb | Null |
+| ItemClicked | Raised when a user interaction causes a jump to the CurrentItem.  | N/A |
+
+Note that V2 of Breadcrumb will add flyouts from chevrons to view children of a node. Additional properties will be added to the API to enable this functionality. 
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+## API
+~~~~
+
+runtimeclass BreadcrumbItemClickedEventArgs
+{
+    Object Item { get; };
+}
+
+unsealed runtimeclass BreadcrumbItem : Windows.UI.Xaml.Controls.ContentControl
+{
+    BreadcrumbItem();
+}
+
+unsealed runtimeclass BreadcrumbLayout : NonVirtualizingLayout
+{
+    BreadcrumbLayout();
+}
+
+[MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+[MUX_PROPERTY_CHANGED_CALLBACK_METHODNAME("OnPropertyChanged")]
+unsealed runtimeclass Breadcrumb : Windows.UI.Xaml.Controls.Control
+{
+    Breadcrumb();
+
+    Object DropdownItemTemplate{ get; set; };
```

Comment:
What is this for and how is it used?

Reply:
The ellipses button when clicked shows a flyout with all the crumbled nodes. This is the template to be used for each item in that flyout.

Reply:
I see. We should explain that above. So in the future there might also be a dropdown on each visible button? Do we need to distinguish the names?

Reply:
I'm not sure if there is a scenario where we need to distinguish between the two. @kat-y ?

Reply:
I don't see a scenario where the flyout of crumbled nodes should be distinct from the (in the future) flyout from chevrons to show children nodes. While the hierarchy structure is different, they are both flyouts from Breadcrumb and consistency here is good.

Reply:
We just need to explain what it is then.

Reply:
I'm adding entries to the API pages section for DropDownItemTemplate and ItemTemplate